packages/clerk-js/src/ui/elements/ContentPage.tsx

```diff
@@ -27,7 +28,7 @@ export const ContentPage = (props: PageProps) => {
       <Header.Root>
         {Breadcrumbs && (
           <Breadcrumbs
-            title={headerTitle}
+            title={breadcrumbTitle ?? headerTitle}
```

Comment:
Do we care about falsy values here? If not, let's prefer ||

Reply:
Again, correct. thanks